src/locales/en.json

```diff
@@ -1426,6 +1432,7 @@
 "label.portableipaddress": "Portable IPs",
 "label.portforwarding": "Port Forwarding",
 "label.powerstate": "Power State",
+"label.preferred": "Prefered",
```

Comment:
Where is this used?

Reply:
https://github.com/apache/cloudstack-primate/blob/master/src/views/offering/AddComputeOffering.vue#L434

Reply:
Sorry older master, that's why I didn't catch it with git grep